docs/api-reference.md

```diff
@@ -483,3 +483,79 @@ type CodeBlock = Omit<CodePaneProps, 'children'> & {
   descriptionProps?: ComponentProps<typeof Text>;
 }
 ```
+
+### `SlideLayout.HorizontalImage`
+
+A layout with one image per slide: a horizontal/landscape image and an optional title and/or description.
+[alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)
+
+| Props               | Type                                    | Required | Example                                                          |
+|---------------------|-----------------------------------------|----------|------------------------------------------------------------------|
+| `...slideProps`     | [Slide Props](#slide)                   | ❌        |                                                                           |
+| `src`               | `string`                                | ✅        | `https://raw.githubusercontent.com/FormidableLabs/dogs/main/src/fred.jpg` |
+| `alt`               | `string`                                | ✅        | `Fred the dog looking at the camera`                                      |
+| `imgProps`          | `ImgHTMLAttributes<HTMLImageElement>`   | ❌        | `{ style: { objectFit: 'contain' } }`                                     |
+| `imgContainerProps` | `ComponentProps<typeof FlexBox>`        | ❌        | `{ style: { border: '8px solid white' } }`                                |
+| `title`             | `string | ReactNode`                    | ❌        | `Fred is a 100% pure bred good boy!`                                      |
```

Comment:
Funky! This is so perplexing to me. I don't _feel_ like I should have to escape the `|` here, haha. 

I will try to figure this out, but I'm not too worried about it since this documentation is getting a face lift in (another PR which moves Slide Layout docs to their own page)[https://github.com/FormidableLabs/spectacle/pull/1237] and I don't see the same issue happening there.

Reply:
> and I don't see the same issue happening there.

oops, I was wrong about this. Following Chris's suggestion and escaping the `|`. 👍